Examples/zib-Pregnancy-Duration-01.xml

```diff
@@ -49,7 +49,9 @@
     <valueQuantity>
         <value value="24"/>
         <unit value="wk"/>
-        <system value="http://unitsofmeasure.org"/>
+        <!-- OID for unitofmeasure -->
+        <system value="urn:oid:2.16.840.1.113883.6.8"/>
+        <!-- <system value="http://unitsofmeasure.org"/>  -->
```

Comment:
http://unitsofmeasure.org staat ertussen op https://www.hl7.org/fhir/stu3/terminologies-systems.html, dus wat mij betreft moet dit teruggedraaid worden, moeten we de fout maar even accepteren totdat https://bits.nictiz.nl/browse/MM-1845 opgelost is.

Reply:
In overleg akkoord. We zetten hem terug.